rotkehlchen/tasks/manager.py

```diff
@@ -707,6 +711,29 @@ def _maybe_update_morpho_vaults(self) -> Optional[list[gevent.Greenlet]]:
 
         return None
 
+    def _maybe_update_aura_pools(self) -> Optional[list[gevent.Greenlet]]:
+        with self.database.conn.read_ctx() as cursor:
+            if (
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.ETHEREUM)) == 0 and  # noqa: E501
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.BASE)) == 0 and  # noqa: E501
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.OPTIMISM)) == 0 and  # noqa: E501
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.POLYGON_POS)) == 0 and  # noqa: E501
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.ARBITRUM_ONE)) == 0 and  # noqa: E501
+                len(self.database.get_single_blockchain_addresses(cursor, SupportedBlockchain.GNOSIS)) == 0  # noqa: E501
+            ):
+                return None
+
+        return [self.greenlet_manager.spawn_and_track(
+                after_seconds=None,
+                task_name=f'Update Aura pools for {chain.to_name()}',
+                exception_is_error=False,
+                method=self.query_aura_pools,
+                evm_inquirer=self.chains_aggregator.get_evm_manager(chain).node_inquirer,  # type: ignore[arg-type]  # chain id is valid
+            )
+            for chain in CHAIN_ID_TO_BOOSTER_ADDRESSES
+            if should_update_protocol_cache(CacheType.AURA_POOLS, (chain.to_name(),)) is True
```

Comment:
Question for here and in the place you save the cahce. This is not for user consumption. Can't you just use the chain id? Won't it store less bytes than the name in the DB?